docs/articles/retrieval_augmented_generation_eval_qdrant_arize.md

```diff
@@ -0,0 +1,519 @@
+
+This includes tracing, prompt iteration, search and retrieval, [evaluating specific tasks](https://arize.com/blog-course/llm-evaluation-the-definitive-guide/), and ultimately improving your LLM through fine-tuning.
+
+This article covers **Phoenix**, which is the [open-source](https://github.com/Arize-ai/phoenix) solution.
+
+The complexity that is involved in building an LLM application is why observability is so important. Each step of the response generation process needs to be monitored, evaluated and tuned to provide the best possible experience. Not only that, certain tradeoffs might need to be made to optimize for speed, cost, or accuracy. In the context of LLM applications, we need to be able to understand the internal state of the system by examining telemetry data such as LLM Traces.
+
+**Traces** are made up of a sequence of `spans`. A span represents a unit of work or operation (think a span of time). It tracks specific operations that a request makes, painting a picture of what happened during the time in which that operation was executed.
+
+LLM Tracing supports the following span kinds:
+
+![../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/arize_spans_types.png](../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/arize_spans_types.png)
+
+By capturing the building blocks of your application while it's running, Phoenix can provide a more complete picture of the inner workings of your application.
+
+**Evaluating a Naive RAG using Phoenix**
+
+To illustrate this, let's look at an example Naive RAG application similar to the one we built in our [previous article]([https://superlinked.com/vectorhub/articles/retrieval-augmented-generation-eval-qdrant-ragas](https://superlinked.com/vectorhub/articles/retrieval-augmented-generation-eval-qdrant-ragas)) and evaluate it using Phoenix.
+
+We will use the collection name as below :
+
+```python
+## Collection Name 
+COLLECTION_NAME = "qdrant_docs_arize_dense"
+```
+
+to index our Naive RAG documents as below :
+
+```python
+## Initializing the space to work with llama-index and related settings
+import llama_index
+from llama_index.core import Settings
+from llama_index.vector_stores.qdrant import QdrantVectorStore
+from phoenix.trace import suppress_tracing
+## Uncomment it if you'd like to use FastEmbed instead of OpenAI
+## For the complete list of supported models,
+##please check https://qdrant.github.io/fastembed/examples/Supported_Models/
+from llama_index.embeddings.fastembed import FastEmbedEmbedding
+
+vector_store = QdrantVectorStore(client=client, collection_name=COLLECTION_NAME)
+
+storage_context = StorageContext.from_defaults(vector_store=vector_store)
+
+## Uncomment if using FastEmbed
```

Comment:
uncomment what?

Reply:
As the Fastembed is used by default , I believe for the snippets , I will leave only for the ones for OpenAI.